config/environments/production.rb

```diff
@@ -76,4 +76,6 @@
 
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
+  config.action_mailer.default_url_options = { :host => 'immense-oasis-2395.heroku.com' }
```

Comment:
Line is too long. [89/80]<br>Use the new Ruby 1.9 hash syntax.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.